src/action_handle_blogger.py

```diff
@@ -173,26 +174,36 @@ def _interact_with_user(device,
     if not profile_filter.check_profile(device, username):
         return False, False
 
-    if likes_count > 12:
+    skip_user_random = (random.randint(1,100))
+    if (skip_user_random <= 10):
+        print(COLOR_OKGREEN + "Random Skip user." + COLOR_ENDC)
+        return False, False
+
+    likes_count_random = likes_count + random.randint(0, 2)
```

Comment:
I think it's better to support "intervals" for all values that we pass to the script. E.g.
**--likes-count 3-6**
**--interactions-count 20-80**
etc.

What do you think?
Aren't you going to do it in this way? Just asking to not do the same job.

Reply:
Yep indeed it may be better to do it like that, in this way everyone can make their choice.
And what do you think of the option that allows you to skip an account during the like process?

Reply:
> And what do you think of the option that allows you to skip an account during the like process?

Isn't that enough that it often skips accounts that have too few photos?